examples/DCCT_plotting.py

```diff
@@ -0,0 +1,29 @@
+"""plotting with DCCT plot
+==========================
+
+This example creates a fake inversion recovery
+data set and plots the data with coherence
+transfer pathways in a reasonable way using
+DCCT plotting."""
+from pyspecdata import * 
+import numpy as np
+from pyspecdata.plot_funcs.DCCT import DCCT
+fl=figlist_var()
+t_axis = nddata(r_[0:2:256j],'t2')
+vd = nddata(r_[0,0.05,0.1,0.15,0.2,0.3,0.4,0.6,0.8,1.0,2.0,3.0,5.0],'vd')
```

Comment:
unrelated

Reply:
also, this has a quit statement, which breaks sphinx, which means that you couldn't have run sphinx to check!!